docs/user-guide/external.md

```diff
@@ -61,12 +61,17 @@ apiVersion: vpc.githedgehog.com/v1beta1
 kind: External
 metadata:
   name: default--5835
+  annotations:
+    type.hhfab.githedgehog.com: hw  # Tells Fabric that this is a physical device
```

Comment:
there is no need to mention this annotation - it's only used for the hybrid vlab which isn't documented in public docs anyways

Reply:
OK, but this documentation is not only public facing, it's also the first reference for us internally. I expect a new hire to look at this for guidance, and without these annotations `hhfab vlab up` will fail on start in an hybrid vlab, so isn't it worth documenting?

Reply:
if we're documenting hybrid vlab it should be clearly separate from any other docs as it's a unique case only for the hybrid vlab. maybe we can introduce a separate section about it